extensions/extensions-helm/aissemble-spark-operator-chart/values.template.yaml

```diff
@@ -56,4 +61,8 @@ spark-operator:
 
     sparkoperator:
       # -- Optional name for the operator service account
-      name: "sparkoperator"
\ No newline at end of file
+      name: "sparkoperator"
+
+rbac:
+  # -- Set to True in order to enable overriding with our own RBAC template
+  createClusterRole: True
```

Comment:
S: There was also some documentation in the README about this custom RBAC that was removed.  Should probably add that back as well.

Reply:
Re-added the rbac.yaml information into the README.